Resources/Prototypes/_Impstation/Entities/Mobs/Species/Visitors/ants.yml

```
@@ -0,0 +1,257 @@
+- type: entity
+  save: false
+  name: Urist McAnt
+  parent: BaseMobSpeciesOrganic
+  id: BaseMobAnt
+  abstract: true
+  components:
+  - type: Hunger
+  - type: Thirst
+  - type: Icon
+    sprite: _Impstation/Mobs/Species/Visitors/Ants/parts.rsi
+    state: full
+  - type: Body
+    prototype: Ant
+    requiredLegs: 2
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+    - id: FoodMeatMeatball #meatball until someone makes ant meat
+      amount: 1 
+  - type: Fixtures
+    fixtures: # TODO: This needs a second fixture just for mob collisions.
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.05
+        # they r smaller #they r smaller (2) #they r smaller (3)
+        density: 10
+        restitution: 0.0
+        mask: 
+        - SmallMobMask
+        layer: #they can go under doors, in exchange for this they die in a single hit
+        - SmallMobLayer
+  - type: Vocal
+    sounds:
+      Male: UnisexAnt
+      Female: UnisexAnt
+      Unsexed: UnisexAnt
+  - type: Damageable
+    damageContainer: Biological
+    damageModifierSet: Ant
```

```suggestion
  - type: Damageable
    damageContainer: Biological
    damageModifierSet: Ant
  - type: MobThresholds
    thresholds:
      0: Alive
      5: Critical
      10: Dead
```
Try this instead of the modifier set, just for consistency. again its probably worth making them resistant to spacing a little unless thats what youre going for.

You could also omit the critical threshold if you just want them to instantly die